lib/p2p/Pool.ts

```diff
@@ -382,7 +382,12 @@ class Pool extends EventEmitter {
    * @param nodePubKey The node pub key of the node for which to get reputation information
    * @return true if the specified node exists and the event was added, false otherwise
    */
-  public getNodeReputation = async (nodePubKey: string): Promise<NodeReputationInfo> => {
+  public getNodeReputation = async (nodeIdentifier: string): Promise<NodeReputationInfo> => {
+    let nodePubKey = nodeIdentifier;
+    if (nodeIdentifier.length !== 66) {
+      const tmp = this.resolveAlias(nodeIdentifier);
+      if (tmp) { nodePubKey = tmp; }
```

Comment:
This line seems mistaken - if `resolveAlias` returns undefined, then we don't update `nodePubKey` and it's value stays as the alias which is not the expected value going forward. I think this is where we'd need something like a `UNKNOWN_ALIAS` error as I also touched on above.